backend/setupTests.ts

```diff
@@ -0,0 +1,11 @@
+import { beforeAll, afterAll } from '@jest/globals';
+import { DatabaseService } from './src/services';
+
+beforeAll(async () => {
```

Comment:
TODO: Remove because `DatabaseService.initDatabase()` and `DatabaseService.closeDatabase()` are already done inside the `main()` server code.